pom.xml

```diff
@@ -198,6 +197,12 @@
                 <artifactId>json</artifactId>
                 <version>20090211</version>
             </dependency>
```

Comment:
This is a comment on the closest change.  Line 200 through 205, which I think needs to be in bom/pom.xml.  Am I missing something on github? I don't believe there is a way to select a group of lines....